client/state/sites/selectors/has-sites-as-landing-page.ts

```diff
@@ -3,14 +3,13 @@ import type { AppState } from 'calypso/types';
 
 export const SITES_AS_LANDING_PAGE_PREFERENCE = 'sites-landing-page';
 
-export const SITES_AS_LANDING_PAGE_DEFAULT_VALUE = {
-	useSitesAsLandingPage: false,
-	updatedAt: 0,
-};
-
 export const hasSitesAsLandingPage = ( state: AppState ): boolean => {
-	const { useSitesAsLandingPage } =
-		getPreference( state, SITES_AS_LANDING_PAGE_PREFERENCE ) ?? SITES_AS_LANDING_PAGE_DEFAULT_VALUE;
-
-	return useSitesAsLandingPage;
+	const { useSitesAsLandingPage = false } = getPreference(
+		state,
+		SITES_AS_LANDING_PAGE_PREFERENCE
+	);
+	if ( useSitesAsLandingPage ) {
+		return useSitesAsLandingPage;
+	}
+	return false;
```

Comment:
Same suggestion here.

```suggestion
	return !! useSitesAsLandingPage;
```